diff --git a/src/libsodium/test/default/Makefile.am b/src/libsodium/test/default/Makefile.am
index 96b677e..b9d726b 100644
--- a/src/libsodium/test/default/Makefile.am
+++ b/src/libsodium/test/default/Makefile.am
@@ -45,7 +45,6 @@
  	onetimeauth7.exp \
  	pwhash_argon2i.exp \
  	pwhash_argon2id.exp \
- 	pwhash_scrypt.exp \
  	pwhash_scrypt_ll.exp \
  	randombytes.exp \
  	scalarmult.exp \
@@ -118,7 +117,6 @@
  	onetimeauth7.res \
  	pwhash_argon2i.res \
  	pwhash_argon2id.res \
- 	pwhash_scrypt.res \
  	pwhash_scrypt_ll.res \
  	randombytes.res \
  	scalarmult.res \
@@ -192,7 +190,6 @@
  	onetimeauth7.final \
  	pwhash_argon2i.final \
  	pwhash_argon2id.final \
- 	pwhash_scrypt.final \
  	pwhash_scrypt_ll.final \
  	randombytes.final \
  	scalarmult.final \
@@ -261,7 +258,6 @@
  	onetimeauth7.nexe \
  	pwhash_argon2i.nexe \
  	pwhash_argon2id.nexe \
- 	pwhash_scrypt.nexe \
  	pwhash_scrypt_ll.nexe \
  	randombytes.nexe \
  	scalarmult.nexe \
@@ -504,9 +500,6 @@
  pwhash_argon2id_SOURCE    = cmptest.h pwhash_argon2id.c
  pwhash_argon2id_LDADD     = $(TESTS_LDADD)
  
- pwhash_scrypt_SOURCE      = cmptest.h pwhash_scrypt.c
- pwhash_scrypt_LDADD       = $(TESTS_LDADD)
- 
  pwhash_scrypt_ll_SOURCE   = cmptest.h pwhash_scrypt_ll.c
  pwhash_scrypt_ll_LDADD    = $(TESTS_LDADD)
  
@@ -593,7 +586,6 @@
  
  if !MINIMAL
  TESTS_TARGETS += \
- 	pwhash_scrypt \
  	pwhash_scrypt_ll \
  	siphashx24 \
  	xchacha20
diff --git a/src/libsodium/test/default/Makefile.in b/src/libsodium/test/default/Makefile.in
index 1cbecb9..822028b 100644
--- a/src/libsodium/test/default/Makefile.in
+++ b/src/libsodium/test/default/Makefile.in
@@ -94,7 +94,6 @@ host_triplet = @host@
  check_PROGRAMS = $(am__EXEEXT_3)
  TESTS = $(am__EXEEXT_3)
  @MINIMAL_FALSE@am__append_2 = \
- @MINIMAL_FALSE@	pwhash_scrypt \
  @MINIMAL_FALSE@	pwhash_scrypt_ll \
  @MINIMAL_FALSE@	siphashx24 \
  @MINIMAL_FALSE@	xchacha20
@@ -121,8 +120,9 @@ CONFIG_CLEAN_VPATH_FILES =
  @EMSCRIPTEN_FALSE@@NATIVECLIENT_FALSE@am__EXEEXT_1 =  \
  @EMSCRIPTEN_FALSE@@NATIVECLIENT_FALSE@	sodium_utils2$(EXEEXT) \
  @EMSCRIPTEN_FALSE@@NATIVECLIENT_FALSE@	sodium_utils3$(EXEEXT)
- @MINIMAL_FALSE@am__EXEEXT_2 = pwhash_scrypt$(EXEEXT) \
- @MINIMAL_FALSE@	pwhash_scrypt_ll$(EXEEXT) siphashx24$(EXEEXT) \
+ @MINIMAL_FALSE@am__EXEEXT_2 = \
+ @MINIMAL_FALSE@ pwhash_scrypt_ll$(EXEEXT) \
+ @MINIMAL_FALSE@ siphashx24$(EXEEXT) \
  @MINIMAL_FALSE@	xchacha20$(EXEEXT)
  am__EXEEXT_3 = aead_aes256gcm$(EXEEXT) aead_chacha20poly1305$(EXEEXT) \
  	aead_xchacha20poly1305$(EXEEXT) auth$(EXEEXT) auth2$(EXEEXT) \
@@ -274,9 +274,6 @@ pwhash_argon2i_DEPENDENCIES = $(TESTS_LDADD)
  pwhash_argon2id_SOURCES = pwhash_argon2id.c
  pwhash_argon2id_OBJECTS = pwhash_argon2id.$(OBJEXT)
  pwhash_argon2id_DEPENDENCIES = $(TESTS_LDADD)
- pwhash_scrypt_SOURCES = pwhash_scrypt.c
- pwhash_scrypt_OBJECTS = pwhash_scrypt.$(OBJEXT)
- pwhash_scrypt_DEPENDENCIES = $(TESTS_LDADD)
  pwhash_scrypt_ll_SOURCES = pwhash_scrypt_ll.c
  pwhash_scrypt_ll_OBJECTS = pwhash_scrypt_ll.$(OBJEXT)
  pwhash_scrypt_ll_DEPENDENCIES = $(TESTS_LDADD)
@@ -403,7 +400,7 @@ SOURCES = aead_aes256gcm.c aead_chacha20poly1305.c \
  	generichash.c generichash2.c generichash3.c hash.c hash3.c \
  	kdf.c keygen.c kx.c metamorphic.c misuse.c onetimeauth.c \
  	onetimeauth2.c onetimeauth7.c pwhash_argon2i.c \
- 	pwhash_argon2id.c pwhash_scrypt.c pwhash_scrypt_ll.c \
+ 	pwhash_argon2id.c pwhash_scrypt_ll.c \
  	randombytes.c scalarmult.c scalarmult2.c scalarmult5.c \
  	scalarmult6.c scalarmult7.c secretbox.c secretbox2.c \
  	secretbox7.c secretbox8.c secretbox_easy.c secretbox_easy2.c \
@@ -419,7 +416,7 @@ DIST_SOURCES = aead_aes256gcm.c aead_chacha20poly1305.c \
  	generichash.c generichash2.c generichash3.c hash.c hash3.c \
  	kdf.c keygen.c kx.c metamorphic.c misuse.c onetimeauth.c \
  	onetimeauth2.c onetimeauth7.c pwhash_argon2i.c \
- 	pwhash_argon2id.c pwhash_scrypt.c pwhash_scrypt_ll.c \
+ 	pwhash_argon2id.c pwhash_scrypt_ll.c \
  	randombytes.c scalarmult.c scalarmult2.c scalarmult5.c \
  	scalarmult6.c scalarmult7.c secretbox.c secretbox2.c \
  	secretbox7.c secretbox8.c secretbox_easy.c secretbox_easy2.c \
@@ -862,7 +859,6 @@ EXTRA_DIST = \
  	onetimeauth7.exp \
  	pwhash_argon2i.exp \
  	pwhash_argon2id.exp \
- 	pwhash_scrypt.exp \
  	pwhash_scrypt_ll.exp \
  	randombytes.exp \
  	scalarmult.exp \
@@ -935,7 +931,6 @@ DISTCLEANFILES = \
  	onetimeauth7.res \
  	pwhash_argon2i.res \
  	pwhash_argon2id.res \
- 	pwhash_scrypt.res \
  	pwhash_scrypt_ll.res \
  	randombytes.res \
  	scalarmult.res \
@@ -1008,7 +1003,6 @@ DISTCLEANFILES = \
  @NATIVECLIENT_TRUE@	onetimeauth7.final \
  @NATIVECLIENT_TRUE@	pwhash_argon2i.final \
  @NATIVECLIENT_TRUE@	pwhash_argon2id.final \
- @NATIVECLIENT_TRUE@	pwhash_scrypt.final \
  @NATIVECLIENT_TRUE@	pwhash_scrypt_ll.final \
  @NATIVECLIENT_TRUE@	randombytes.final \
  @NATIVECLIENT_TRUE@	scalarmult.final \
@@ -1077,7 +1071,6 @@ DISTCLEANFILES = \
  @NATIVECLIENT_TRUE@	onetimeauth7.nexe \
  @NATIVECLIENT_TRUE@	pwhash_argon2i.nexe \
  @NATIVECLIENT_TRUE@	pwhash_argon2id.nexe \
- @NATIVECLIENT_TRUE@	pwhash_scrypt.nexe \
  @NATIVECLIENT_TRUE@	pwhash_scrypt_ll.nexe \
  @NATIVECLIENT_TRUE@	randombytes.nexe \
  @NATIVECLIENT_TRUE@	scalarmult.nexe \
@@ -1211,8 +1204,6 @@ pwhash_argon2i_SOURCE = cmptest.h pwhash_argon2i.c
  pwhash_argon2i_LDADD = $(TESTS_LDADD)
  pwhash_argon2id_SOURCE = cmptest.h pwhash_argon2id.c
  pwhash_argon2id_LDADD = $(TESTS_LDADD)
- pwhash_scrypt_SOURCE = cmptest.h pwhash_scrypt.c
- pwhash_scrypt_LDADD = $(TESTS_LDADD)
  pwhash_scrypt_ll_SOURCE = cmptest.h pwhash_scrypt_ll.c
  pwhash_scrypt_ll_LDADD = $(TESTS_LDADD)
  randombytes_SOURCE = cmptest.h randombytes.c
@@ -1477,10 +1468,6 @@ pwhash_argon2id$(EXEEXT): $(pwhash_argon2id_OBJECTS) $(pwhash_argon2id_DEPENDENC
  	@rm -f pwhash_argon2id$(EXEEXT)
  	$(AM_V_CCLD)$(LINK) $(pwhash_argon2id_OBJECTS) $(pwhash_argon2id_LDADD) $(LIBS)
  
- pwhash_scrypt$(EXEEXT): $(pwhash_scrypt_OBJECTS) $(pwhash_scrypt_DEPENDENCIES) $(EXTRA_pwhash_scrypt_DEPENDENCIES) 
- 	@rm -f pwhash_scrypt$(EXEEXT)
- 	$(AM_V_CCLD)$(LINK) $(pwhash_scrypt_OBJECTS) $(pwhash_scrypt_LDADD) $(LIBS)
- 
  pwhash_scrypt_ll$(EXEEXT): $(pwhash_scrypt_ll_OBJECTS) $(pwhash_scrypt_ll_DEPENDENCIES) $(EXTRA_pwhash_scrypt_ll_DEPENDENCIES) 
  	@rm -f pwhash_scrypt_ll$(EXEEXT)
  	$(AM_V_CCLD)$(LINK) $(pwhash_scrypt_ll_OBJECTS) $(pwhash_scrypt_ll_LDADD) $(LIBS)
@@ -1640,7 +1627,6 @@ distclean-compile:
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/onetimeauth7.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/pwhash_argon2i.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/pwhash_argon2id.Po@am__quote@
- @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/pwhash_scrypt.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/pwhash_scrypt_ll.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/randombytes.Po@am__quote@
  @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/scalarmult.Po@am__quote@
@@ -2355,13 +2341,6 @@ sodium_utils3.log: sodium_utils3$(EXEEXT)
  	--log-file $$b.log --trs-file $$b.trs \
  	$(am__common_driver_flags) $(AM_LOG_DRIVER_FLAGS) $(LOG_DRIVER_FLAGS) -- $(LOG_COMPILE) \
  	"$$tst" $(AM_TESTS_FD_REDIRECT)
- pwhash_scrypt.log: pwhash_scrypt$(EXEEXT)
- 	@p='pwhash_scrypt$(EXEEXT)'; \
- 	b='pwhash_scrypt'; \
- 	$(am__check_pre) $(LOG_DRIVER) --test-name "$$f" \
- 	--log-file $$b.log --trs-file $$b.trs \
- 	$(am__common_driver_flags) $(AM_LOG_DRIVER_FLAGS) $(LOG_DRIVER_FLAGS) -- $(LOG_COMPILE) \
- 	"$$tst" $(AM_TESTS_FD_REDIRECT)
  pwhash_scrypt_ll.log: pwhash_scrypt_ll$(EXEEXT)
  	@p='pwhash_scrypt_ll$(EXEEXT)'; \
  	b='pwhash_scrypt_ll'; \
