diff --git a/Lib/sysconfig.py b/Lib/sysconfig.py
index 9c8350d953..04805f1c5e 100644
--- a/Lib/sysconfig.py
+++ b/Lib/sysconfig.py
@@ -312,12 +312,6 @@ def _generate_posix_vars():
             msg = msg + " (%s)" % e.strerror
         raise IOError(msg)
 
-    # On AIX, there are wrong paths to the linker scripts in the Makefile
-    # -- these paths are relative to the Python source, but when installed
-    # the scripts are in another directory.
-    if _PYTHON_BUILD:
-        vars['LDSHARED'] = vars['BLDSHARED']
-
     # There's a chicken-and-egg situation on OS X with regards to the
     # _sysconfigdata module after the changes introduced by #15298:
     # get_config_vars() is called by get_platform() as part of the
@@ -361,6 +355,12 @@ def _init_posix(vars):
     from _sysconfigdata import build_time_vars
     vars.update(build_time_vars)
 
+    # On AIX, there are wrong paths to the linker scripts in the Makefile
+    # -- these paths are relative to the Python source, but when installed
+    # the scripts are in another directory.
+    if _PYTHON_BUILD:
+        vars['LDSHARED'] = vars['BLDSHARED']
+
 def _init_non_posix(vars):
     """Initialize the module as appropriate for NT"""
     # set basic install directories
