--- a/psutil/arch/aix/net_kernel_structs.h	2020-04-22 14:17:50.000000000 +0200
+++ b/psutil/arch/aix/net_kernel_structs.h	2020-04-22 14:18:13.000000000 +0200
@@ -17,8 +17,8 @@
 #include <sys/socketvar.h>
 #include <sys/protosw.h>
 #include <sys/unpcb.h>
-#include <sys/mbuf_base.h>
+#include <sys/mbuf.h>
 #include <sys/mbuf_macro.h>
 #include <netinet/ip_var.h>
 #include <netinet/tcp.h>
 #include <netinet/tcpip.h>