--- psutil-release-5.6.6/psutil/_psutil_aix.c-orig     2021-08-12 15:08:50.000000000 -0500
+++ psutil-release-5.6.6/psutil/_psutil_aix.c  2021-08-12 15:08:02.000000000 -0500
@@ -117,8 +117,12 @@
          status.pr_stat = SACTIVE;
      } else {
          sprintf(path, "%s/%i/status", procfs_path, pid);
-         if (! psutil_file_to_struct(path, (void *)&status, sizeof(status)))
-             return NULL;
+         if (! psutil_file_to_struct(path, (void *)&status, sizeof(status))) {
+             // Can't access /proc/<pid>/status (eg: access denied)
+             // Continue without the process status
+             PyErr_Clear();
+             status.pr_stat = 0;
+         }
      }
  
      return Py_BuildValue("KKKdiiiK",
